src/python/txtai/pipeline/text/translation.py

```diff
@@ -26,7 +26,7 @@ class Translation(HFModel):
     # Default language detection model
     DEFAULT_LANG_DETECT = "https://dl.fbaipublicfiles.com/fasttext/supervised-models/lid.176.ftz"
```

Comment:
Can we either delete this line or use the constant on line 149?

Reply:
done